articles/howto-build-astatic-site-with-gatsbyjs/index.md

```diff
@@ -0,0 +1,281 @@
+
+Yes, good! The moment has come for us to use it now, but don’t worry we will just be working with the basics here that are pretty easy.
+
+In the index.js file add this piece of code:
+
+```JavaScript
+// index.js code
+import React from "react"
+
+export default function Home() {
+return <div style={{ color: 'tomato' }}>
+    <h1>Hello world!</h1>
+    // Your new paragraph or content
+    <p>Welcome to my first Gatsby site</p>
+</div>
+}
+```
+
+### Linking between pages
+
+Having a one-page static site doesn’t sound cool at all, what if you want to have 2 or more pages? To do this create another page and link them together, so that you can easily navigate through the added pages.
+
+First of all, import a Gatsby react link and add it to the top of the index.js file
+`import { Link } from "gatsby"`.
+
+Secondly, create a .js file with the name **page-2.js** in the pages folder and add this code snippet:
+
+```JavaScript
+//page-2 code
+import React from "react"
+import { Link } from "gatsby"
+
+export default function Home() {
+    return <div style={{ color: 'tomato' }}>
+        <h1>Welcome to Page 2</h1>
+        // To go back to the homepage
+        <Link to="/">Back</Link>
+    </div>
+}
+```
+
+Then finally link it using the Gatsby link by placing this code in the index.js file to connect it to the page-2.js file
+
+```bash
+<Link to=”/page-2/”>Page 2</Link>
+```
+
+### Counter.js file
+
+Here you are going to make the site more interactive by adding a counter to the site where you can click on a **plus** button to increase a number and a **minus** button to decrease a number.
+
+Create a counter.js file in the pages folder and then add this code snippet.
+```JavaScript
+//counter code
+import React from "react"
+
+class Counter extends React.Component {
+    constructor() {
+        super()
+        this.state = { count: 0 }
+    }
+    render() {
+        return <div style={{ color: 'blue' }}>
+            <h1>Counter</h1>
+            <p>current counter: {this.state.count}</p>
+            //plus button
+            <button onClick={() => this.setState({ count: this.state.count + 1 })} style={{ color: 'tomato' }}>PLUS</button>
+            //minus button
+            <button onClick={() => this.setState({ count: this.state.count - 1 })} style={{ color: 'tomato' }}>MINUS</button>
+            <br></br>
+            <br></br>
+            <br></br>
+            // To go back to the homepage
+            <Link to="/">Back</Link>
+        </div>
+    }
+}
+
+export default
+```
+
+### Using React Components
+
+Using react components to build your static site will help you get the job done faster. With react you can make parts of your code as a component, component codes are reusable and can be placed in any of your react code file to be used again.
+
+Yet, I highly recommend you check out the [react component documentation](https://reactjs.org/docs/react-component.html) to gain a better understanding of react components.
```

Comment:
You should still give a better explanation of components for the reader to know what is going on

Reply:
Done